### ISSUE TITLE
ParentCallSpacing should be ignored in match expressions

### ISSUE THREAD

**Issue by dmolineus** [slevomat/coding-standard#1546]
Following example should not trigger the ParentCallSpacing sniff:

```php
return match ($event::class) {
    ProjectFetched::class, ProjectUpdated::class => 'onProjectFetchedOrUpdated',
    default => parent::inflectHandlerMethod($event),
};
```

Adding `T_MATCH_ARROW` here works.

https://github.com/slevomat/coding-standard/blob/2a778611a1f17a70ae2c5426b8b42e5e87bf6eda/SlevomatCodingStandard/Sniffs/Classes/ParentCallSpacingSniff.php#L65

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/13580e240d0bb9190097b4d9151205ea1f5bf035
